include/asterisk/stasis_app.h

```diff
@@ -121,6 +131,12 @@ int stasis_app_register_all(const char *app_name, stasis_app_cb handler, void *d
  */
 void stasis_app_unregister(const char *app_name);
 
+/*!
+ * \brief Unregister a Stasis application and remove all subscriptions.
+ * \param app_name Name of the application to unregister.
+ */
+void stasis_app_unregister_all(const char *app_name);
```

Comment:
I don't think the name of this function reflects what it does, it reads as though it is unregistering all apps despite taking an app name.

Reply:
I didn't like it either but I followed the existing naming pattern for register... stasis_app_register(), stasis_app_register_all().  How about stasis_app_unregister_unsubscribe_all()?